test/xhr.spec.js

```diff
@@ -36,6 +37,20 @@ describe('XHR backend', () => {
       });
     });
 
+    it('should load json5 data', (done) => {
+      backend = new XHR({
```

Comment:
I did `new XHR` here instead of `backend.options.loadPath += '5'` and `backend.options.parse = fn` to see that the constructor worked as intended. Might have this in a `before` block, but did not want to restructure.